src/Pokémon.java

```diff
@@ -0,0 +1,41 @@
+public abstract class Pokémon {
```

Comment:
Goed dat je de class abstract hebt gemaakt. Ik mis echter nog een abstracte methode in deze class.